dmutils/__init__.py

```diff
@@ -4,4 +4,4 @@
 import flask_featureflags  # noqa
 
 
-__version__ = '31.1.0'
+__version__ = '31.1.1'
```

Comment:
Would this maybe be considered a breaking/major change? Previously this would ignore/bypass files that didn't have a custom timestamp; now it will return any file present in the bucket.